FreeRTOS_IP_Timers.c

```diff
@@ -484,3 +484,17 @@ void vIPSetARPResolutionTimerEnableState( BaseType_t xEnableState )
 
 #endif /* ipconfigUSE_DHCP */
 /*-----------------------------------------------------------*/
+
+#if ( ipconfigDNS_USE_CALLBACKS != 0 )
+
+/**
+ * @brief Reload the DNS timer.
+ *
+ * @param[in] ulCheckTime: The reload value.
+ */
+    void vIPReloadDNSTimer( uint32_t ulCheckTime )
+    {
+        prvIPTimerReload( &xDNSTimer, ulCheckTime );
+    }
+#endif /* ipconfigDNS_USE_CALLBACKS != 0 */
```

Comment:
There is already a definition at line 338 (albeit with a different name). I renamed the function to make all reloading functions have similar names.